src/dashboard/apigateway/apigateway/biz/resource_version.py

```diff
@@ -143,7 +143,7 @@ def create_resource_version(cls, gateway: Gateway, data: Dict[str, Any], usernam
                 # TODO: 待 version 改为必填后，下面的 version 赋值去掉
                 "version": data.get("version") or name,
                 "created_time": now,
-                "scheme_version": ResourceVersionSchemaEnum.V2Version.value,
+                "schema_version": ResourceVersionSchemaEnum.V2Version.value,
```

Comment:
should be `ResourceVersionSchemaVersionEnum.V2.value`

Reply:
done